### ISSUE TITLE
Uncertainty mismatch between CHIME catalog and TNS

### ISSUE THREAD

**Issue by 0xCoto** [chime-frb-open-data/community#70]
It appears that certain bursts have different uncertainty values reported for the source coordinates, between the CHIME catalog and the TNS database.

For example, FRB 20190518D has a Dec. error of 38.18 deg according to the TNS (https://www.wis-tns.org/object/20190518D), but 0.24 deg according to the CHIME catalog (http://chime-frb.ca/catalog/FRB20190518D).

RA error also appears off (TNS: 36 arcmin vs CHIME: 19.8 arcmin).

The same seems to be true for other bursts as well. What might have been the cause of such mismatch? I assume the uncertainties reported on the CHIME catalog are the most accurate?

**Comment by emilypetroff** [comment]
Good catch. Yes, the most accurate uncertainties will be those reported in the catalog paper and in the associated data tables on the CHIME/FRB public webpage. I am not sure about the source of the mismatch, but my guess is the following: the final catalog sample was prepared for TNS upload with certain parameters and pushed to the TNS ahead of the catalog release. An additional step of processing was likely done before publication resulting in slightly different values which were updated in the publication-associated data tables but not pushed to the TNS as the events had already been uploaded. 

In cases where there is a mismatch between values on TNS and CHIME/FRB public webpage, the values on the public page will always be the most up-to-date.

**Comment by 0xCoto** [comment]
I see, thanks for the clarification. Do you happen to know whether "repeater of FRB ..." entries might have been affected by the inaccurate uncertainties, or are they derived and reported separately?

I'm currently working on an automated repeater classification/clustering algorithm and want to verify that the TNS list of repeaters is a good reference to have as a "ground truth" baseline. If that's the case, I'd only have to correct the errors based on the CHIME catalog and rerun the analysis.
